SPECS/openblas/openblas.spec

```diff
@@ -20,27 +20,22 @@ Summary:        An optimized BLAS library based on GotoBLAS2
 # The same spec is also used on the EPEL branches, meaninng that some
 # "obsoleted" features are still kept in the spec.
 Name:           openblas
-Version:        0.3.21
-Release:        3%{?dist}
-License:        BSD
+Version:        0.3.26
+Release:        1%{?dist}
+License:        BSD-3-Clause
```

Comment:
Where did you find this License change?

Reply:
I found the License in : https://github.com/OpenMathLib/OpenBLAS/wiki#license
BSD-3-Clause is more specific than BSD, should we make it more precise? 
A reference is Fedora updated the License of openblas while upgrading from 0.3.21 to 0.3.24: https://src.fedoraproject.org/rpms/openblas/c/5123a10ecd1d6c5382ea7bb5d6187a6bca030134